.elasticbeanstalk/config.yml

```diff
@@ -0,0 +1,21 @@
+branch-defaults:
```

Comment:
This file is created by the ElasticBeanstalk CLI when running `eb init` on the project directory. It contains settings used by the `eb` command to connect to the desired AWS EC2 instance.